### ISSUE TITLE
Taking my leave

### ISSUE THREAD

**Issue by Javran** [KC3Kai/KC3Kai#1905]
This issue is just for an announcement and can be closed after few days.

Here I officially announce that I'm leaving from both dev team and translator duty for scn. because I can't find another solution for dealing with a childish and irresponsible developer - I've tried to be reasonable, despite several times end up in anger, but it never worked. and sadly he is the only active one around - with only two people there's no way of telling the other what's a common sense. I'm fed up.

Feel free to remove me from teams as well, I'm pretty much prefer PR approach anyways - given that there's a need.

While I'm leaving there are several maintenance stuff I want to point out as my final contribution:

- whenever any of the remodeling chain changes, overwrite `tools/remodelGroups.json`  in kc3-translations with output of `RemodelDb.dumpRemodelGroups()`
- regarding quotes of scn translation, there is a section on https://github.com/KC3Kai/kc3-translations/tree/master/data/scn about how to update it
- for KanColleHelpers, take a look at https://github.com/Javran/purescript-kancolle-helpers, run `scripts/mkRelease.sh` to generate a new version to overwrite file with the same name under `src/assets/js`, you don't have to do this very often, only when there are changes to battle mechanism or expeditions, after I have adjusted the code accordingly.
- for how I mark node letters, take a look at "maptool" part of https://github.com/Javran/kantour, keep in mind that while edge numbers are 100% correct, node letters are guessed by the program and you might modify some of them to respect the actual letter on map.

Thanks fellows, it was a good time.

**Comment by dragonjet** [comment]
Thank you for all the hard work!

Sorry Im not a good project manager to be able to step in on the discussions, as it has been happening not just this once.

As it makes me sad this has to happen, but this time we should look back and be happy with that we achieved. KC3 has been _one of the_ best tools in Kancolle with your contributions.

I am also leaving myself, as I have posted in an issue thread tagged above. It was a pleasure having to work with you. You might be still making contributions as external PR, but I'm not, so I'm saying this :smile:
